codespell_lib/data/dictionary.txt

```diff
@@ -16364,6 +16366,7 @@ specfy->specify
 specfying->specifying
 speciafied->specified
 specialiced->specialised, specialized,
+specialise->specialize
```

Comment:
British English, remove it please and add it to the issue.

Reply:
Done